### PR TITLE
fix: let formatter function run even when value is falsy

### PR DESCRIPTION
**Pull request by jeremy0dell**
Changed the logical AND operators (&&) added in c6cf8dc to nullish coalescing operators (??) to allow formatter function to run even when value is falsy. 0 is a falsy value that still ought to let the formatter function run

### CODE REVIEW

File: src/component/DefaultTooltipContent.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class DefaultTooltipContent<TValue extends ValueType, TName extends NameT
         };
         const finalFormatter = entry.formatter || formatter || defaultFormatter;
         let { value, name } = entry;
-        if (finalFormatter && value && name) {
+        if (finalFormatter ?? value ?? name) {
```
Comment (mguida22) [inline src/component/DefaultTooltipContent.tsx:82]: I believe this should check if the value and name are not null or undefined? With `??` value and name can be undefined, which I don't think is the intention here.

```suggestion
        if (finalFormatter && value != null && name != null) {
```

**Review by mguida22 — COMMENTED** [review]
Thanks for putting up this PR! We're running into the same issue right now, it'd be great to get this fix in.

File: src/component/DefaultTooltipContent.tsx (hunk 1 above)
Comment (jeremy0dell) [reply src/component/DefaultTooltipContent.tsx:82]: You're right! I'm still getting used to the `??` operator. Committing this change

**Comment by jeremy0dell** [comment]
@mguida22 Maybe we could run the test suite and get this into master?

**Comment by mguida22** [comment]
I don't have permissions to run anything here, I just ran into the same problem and saw your open PR 😬

**Comment by jeremy0dell** [comment]
Ah. @arcthur thoughts?

**Pull request merged**